emulated_hue/api.py

```diff
@@ -103,7 +103,6 @@ def __init__(self, hue):
         """Initialize with Hue object."""
         self.streaming_api = None
         self.config = hue.config
```

Comment:
Since we removed the instance variable hass, would we also want to remove config as well since its simply a reference to hue.config?

Reply:
Let's do that in a next refactor. Step by step ;-)